### DISCUSSION TITLE
Blueretro mapping for Bleemcast on Dreamcast console

### DISCUSSION THREAD

**Discussion by Tredes** [darthcloud/BlueRetro#908] [Q&A] [answered]
Thank you for your great work and to request that you include a new default configuration in the Blueretro DC options, which would be directed to the Bleemcast emulator for both Gdemu and physical disks. I share with you a mapping that can perhaps be improved to have the Playstation buttons correctly located. 
Since this mapping is inspired by Bleempad, and since the VMUs for DC are implemented. Will it also be a possibility to copy a Memory Card PS1 into BlueretroDC and save games as well?
thanks for your attention.
https://www.mediafire.com/view/0cpfp071fb9kh7j/Bleem_DC_Blueretro.jpg/file

**Comment by darthcloud** [comment, marked as answer]
Thanks for this.

You think you can open a PR to add it to the web interface?
https://github.com/darthcloud/BlueRetroWebCfg/tree/master/map

**Reply by Tredes** [reply]
Sorry for the delay, and I hope I've handled the entire PR application process correctly. Thank you for your trust, and I hope this contribution is well received.

**Comment by Tredes** [comment]
Good morning. While testing the preset configuration in the "Bleemcast PS1 Dreamcast with hotkey" web interface, there are two assignments to make the preset work automatically:

"PAD_MT" (PS button) corresponds to "PAD_RB_RIGHT" (button B or circle). The similar assignments for buttons A, X, and Y are correct.

"PAD_RM" (R2 trigger) corresponds to "PAD_LD_RIGHT" (left directional pad right).

Please review it. I'll be attentive if you give me instructions on how to proceed if I need to modify any parameters.
Thank you very much.

Ya pude corregirlo y enviado
